packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -84,6 +84,13 @@ export interface DatabaseClusterProps {
    */
   readonly defaultDatabaseName?: string;
 
+  /**
+   * Indicates whether the DB cluster should have deletion protection enabled.
+   *
+   * @default true
+   */
+  readonly deletionProtection?: boolean;
```

Comment:
According to CloudFormation [docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbcluster.html#cfn-rds-dbcluster-deletionprotection), the default is false, so unless we set it to true it will be false.